docs/source/How_to_install_liminal_in_airflow_on_kubernetes.md

```diff
@@ -0,0 +1,160 @@
+|[Airflow Helm Chart][airflowChart] | 7.14.3 |
+
+#### You may need to add the Repo
+
+```sh
+helm repo add airflow-stable https://airflow-helm.github.io/charts
+helm repo update
+```
+
+## Get Started
+
+### Scripted Installation
+
+[Use `install_liminal_in_airflow_on_kubernetes.sh`][liminal-installation-script] to install liminal in Airflow.
+* Move to the [example repository project][liminal-getting-started-project]
+* Copy the [script][liminal-installation-script] to the [example repository project][liminal-getting-started-project] and run it
+
+![](assets/install_liminal_in_airflow_on_kubernetes.gif)
+
+### Manual Installation
+
+Here are a couple of steps of how to install liminal in Airflow on Kubernetes:
+* [Find Airflow components](#Find-Airflow-pod-names)
+* [Install liminal in the components](#Install-liminal-in-Airflow-pods)
+* [Setting up liminal](#Setting-up-liminal)
+
+#### Find Airflow pod names:
+
+```sh
+echo -n "Please enter a namespace: "
+read namespace
+components=("web" "scheduler" "worker")
+namespace=${namespace}
+podNames=()
+for component in ${components[@]}
+do
+        podNames+="$(kubectl get pod -n ${namespace} -l "app=airflow,component=${component}" --no-headers -o custom-columns=":metadata.name")\n"
+
+done
+
+echo -e "The following Airflow pods are:\n${podNames}"
+```
+#### Install liminal in Airflow pods:
+
+```sh
+IFS=$'\n'|''; set -f; podNames=( ${podNames} )
+webPodName=${podNames[${#podNames[@]} - 1]}
+
+for podName in `echo -e ${podNames[@]}`
+do
+        echo "Installing liminal in pod: ${podName}"
+        kubectl exec -it -n ${namespace} ${podName} -- bash -c 'pip install --user apache-liminal'
+done
+```
+
+#### Setting up liminal:
+There are a couple of ways to deploy Yamls in Airflow.
+1. Mount your Amazon EFS file system on a Linux instance:
+    ```
+    sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport <EFS_ID>:/ /opt/airflow/dags
+    ```
+2. Use gitsync function in order to sync Yamls in the Amazon EFS file system. In this case, we chose to use a kubernetes job:
+    ```
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      namespace: production
+      name: airflower-production-git
+    spec:
+      activeDeadlineSeconds: 60
+      backoffLimit: 1
+      template:
+        spec:
+          containers:
+          - name: git
+            image: alpine/git
+            command: ["/bin/sh","-c"]
+            args: ["cd /dags; git fetch && git reset --hard origin/master"]
+            volumeMounts:
+            - mountPath: /dags
+              name: dags
+          volumes:
+          - name: dags
+            persistentVolumeClaim:
+              claimName: airflow-production
+          restartPolicy: Never
+    ```
+3. Deploy the Yamls by using kubectl in order to copy the Yamls to the mounted file system. In this case, we chose to use kubectl to copy the Yamls to the mounted EFS:
```

Comment:
Again, in terms of completeness - you are right, it can also be done in this way.
however we need to focus on our recommended way, for which we asked the user to set up EFS

you can mention it in the previous section in passing:
There are other ways to deploy into Aiflow.
You can poll a git repo from inside airflow and copy the files in pull.
Alternatively, you can do something similar to how we installed liminal on the pods, and iterate on them, copying the file into each.

However, to use native Liminal tools, we recommend the following approach....